index.d.ts

```diff
@@ -1036,7 +1036,7 @@ declare module 'binance-api-node' {
 
   export interface SymbolMinNotionalFilter {
     filterType: SymbolFilterType.MIN_NOTIONAL
-    notional: string
+    minNotional: string
```

Comment:
@bennycode what's the consensus on this one? Pretty sure there's a lot of back and forth on this which might be due to the fact the field changes depending on which endpoint is hit

Reply:
Yepp, that's why there is now a `FuturesSymbolMinNotionalFilter` and a `SymbolMinNotionalFilter`. 👍

Reply:
Oh actually you were part of the original convo, my bad

Reply:
Gotcha thank you for the confirmation